axelrod/game.py

```diff
@@ -16,9 +18,77 @@ class Game(object):
         The numerical score attribute to all combinations of action pairs.
     """
 
-    def __init__(
-        self, r: Score = 3, s: Score = 0, t: Score = 5, p: Score = 1
-    ) -> None:
+    # pylint: disable=invalid-name
+    def __init__(self, A: np.array, B: np.array) -> None:
+        """
+        Creates an asymmetric game from two matrices.
+
+        Parameters
+        ----------
+        A: np.array
+            the payoff matrix for player A.
+        B: np.array
+            the payoff matrix for player B.
+        """
+
+        self.A = A
```

Comment:
Can we add an error / warning if A and B.transpose do not have the same dimensions?